website/src/pages/dictionary/index.js

```diff
@@ -185,41 +196,67 @@ function DataDictionary() {
       dictionary.schemas
         .map((schema) => {
           const { tier, attribute } = searchParams;
-          const filteredFields = schema.fields.filter((field) => {
-            const meta = get(field, 'meta', {});
-            const { primaryId = false, core = false, dependsOn = false } = meta;
-            const required = get(field, 'restrictions.required', false);
-
-            let tierBool = false;
-            let attributeBool = false;
-
-            if (tier === NO_ACTIVE_FILTER && attribute === NO_ACTIVE_FILTER) return true;
-
-            if (
-              (tier === TAG_TYPES.id && primaryId) ||
-              (tier === TAG_TYPES.core && core) ||
-              (tier === TAG_TYPES.extended && !core && !primaryId) ||
-              tier === '' ||
-              tier === NO_ACTIVE_FILTER
-            ) {
-              tierBool = true;
-            }
-
-            if (
-              (attribute === TAG_TYPES.conditional && Boolean(dependsOn)) ||
-              (attribute === TAG_TYPES.required && required) ||
-              attribute === '' ||
-              attribute === NO_ACTIVE_FILTER
-            ) {
-              attributeBool = true;
-            }
-
-            return tierBool && attributeBool;
-          });
+          const filteredFields = schema.fields
+            /**
+             * !!!!!!!!!!!!!!!!!!!!!!
+             * Remove this after demo
+             * !!!!!!!!!!!!!!!!!!!!!!
+             */
+            .map((fields, i) => {
+              if (i === 2) {
+                return { ...fields, ...{ compare: 'addition' } };
+              } else if (i === 3) {
+                return { ...fields, ...{ compare: 'deletion' } };
+              } else if (i === 7) {
+                return { ...fields, ...{ compare: 'update' } };
```

Comment:
should we also just make this uppercase? maybe use `compareFilterTypes.ADDITION` for consistency